twitter.py

```diff
@@ -0,0 +1,87 @@
+"""
+This file gets the mentions on twitter and creates a spotify playlist then retweets it
+back to the user who requested it.
+"""
+import os
+import re
+import tweepy
+from createSpotifyPlaylist import create_spotify_playlist_from_search
+
```

Comment:
missing a space

Reply:
fixed this spacing and other spacing [here](https://github.com/GaelGil/apiPractice/commit/24d5855d83265e21000dd2b174bee22fdb6850ea)